store/adapters/rdbms/drivers/dialect.go

```diff
@@ -42,3 +65,121 @@ type (
 		ExprHandler(*ql.ASTNode, ...exp.Expression) (exp.Expression, error)
 	}
 )
+
+func init() {
+	goqu.SetDefaultPrepared(true)
+}
+
+func IndexFieldModifiers(attr *dal.Attribute, quoteIdent func(i string) string, mm ...dal.IndexFieldModifier) (string, error) {
+	var (
+		modifier string
+		out      = quoteIdent(attr.StoreIdent())
+	)
+
+	for _, m := range mm {
+		switch m {
+		case dal.IndexFieldModifierLower:
+			modifier = "LOWER"
+		default:
+			return "", fmt.Errorf("unknown index field modifier: %s", m)
+		}
+
+		out = fmt.Sprintf("%s(%s)", modifier, out)
+	}
+
+	return out, nil
+}
+
+func OpHandlerIn(d Dialect, n *ql.ASTNode, args ...exp.Expression) (expr exp.Expression, err error) {
+	return opHandlerIn(d, n, false, args...)
+}
+
+func OpHandlerNotIn(d Dialect, n *ql.ASTNode, args ...exp.Expression) (expr exp.Expression, err error) {
+	return opHandlerIn(d, n, true, args...)
+}
+
+func opHandlerIn(d Dialect, n *ql.ASTNode, negate bool, args ...exp.Expression) (expr exp.Expression, err error) {
+	if n.Args[1] != nil && n.Args[1].Meta["dal.Attribute"] != nil && n.Args[1].Meta["dal.Attribute"].(*dal.Attribute).MultiValue {
```

Comment:
can n.Args ever be empty? This would panic then

Reply:
I thought I can rely on `ql` package :)